markdown/solace-pubsub-microgateway/solace-pubsub-microgateway.md

```diff
@@ -0,0 +1,1185 @@
+author: Aaron Lee
+summary: Summary of your codelab that is human readable
+id: solace-pubsub-microgateway
```

Comment:
What are your thoughts on keeping this the same vs. changing it to "event-enable-rest" or something like that to match more of what the title/pattern is?

Reply:
Yeah, I don't mind.

Reply:
Yes I agree. let's start with the "solace-" prefix. I agree it's only in the URL, but that will keep the URL consistent with the name on the website and with the folder/resource names in the git repo so it will make sense if someone else comes to update this. (So please change the folder names & markdown file name as well to match)

Reply:
"solace-event-enable-rest" done.